src/SCS.jl

```diff
@@ -16,7 +16,12 @@ function __init__()
     if is_unix()
         dlopen(Base.liblapack_name, RTLD_LAZY|RTLD_DEEPBIND|RTLD_GLOBAL)
     end
-    if !(vnum.major == 1 && vnum.minor == 1)
+    if is_linux()
+        if !(vnum.major == 1 && vnum.minor == 2)
```

Comment:
So? They shouldn't be *forced* if the package will work fine. This code here is to avoid segfaults with mismatching binary interfaces. It could be worth printing a warning though.